internal/action/generate.go

```diff
@@ -120,7 +120,7 @@ func (s *Action) generateCopyOrPrint(ctx context.Context, c *cli.Context, name,
 	// copy to clipboard if:
 	// - explicitly requested with -c
 	// - autoclip=true, but only if output is not being redirected
-	if IsClip(ctx) || (s.cfg.AutoClip && !ctxutil.IsTerminal(ctx)) {
+	if IsClip(ctx) || (s.cfg.AutoClip && ctxutil.IsTerminal(ctx)) {
```

Comment:
Maybe I should be replacing the content of my new function `out.IsOutputRedirected`  by this then.
I'm not sure how `ctxutil.IsTerminal(ctx)` behaves when we are piping stuff but in a terminal.

Reply:
Now in #2025